service environment variable (#3809)
platformsh/platformsh-docs: sites/friday/src/development/variables/_index.md

So you can use the same codebase and settings for all your environments,
but still adapt the behavior to each environment.

### Service-specific variables

> For each service defined as a relationship to your application, {{% vendor/name %}} automatically generate corresponding environment variables within your application container, with the pattern:
> ```
> {{< variable "<RELATIONSHIP_NAME>" >}}_<KEY>=<VALUE>
> ```

Reviewer: ```suggestion
For each service defined as a relationship to your application, {{% vendor/name %}} automatically generates corresponding environment variables within your application container, in the ${{< variable "RELATIONSHIP_NAME" >}}-_{{< variable "SERVICE_PROPERTY" >}} format.
```
